### DISCUSSION TITLE
Multilingual docs don't work with mike versioning

### DISCUSSION THREAD

**Discussion by berkantsahn** [squidfunk/mkdocs-material#5364] [Q&A]
Hi everyone,

I'm trying to do `mike` versioning with `multiple languages` but i couldn't do it. 
I can do `multi language` and versioning with `mike` separately and they are working.
When doing multi language (2 different languages) I create two different mkdocs.yml (mkdocs.tr.yml and mkdocs.en.yml etc.) files and build the pages according to them but when I add the versioning process, i am having problems about routing and menus are broken

My repo link: https://github.com/berkantsahn/mkdocs-sample

**Comment by squidfunk** [comment]
So this is actually a general question that is not tied to Material for MkDocs. Yes, we recommend using mike on our documentation and we have native support for version selection, but what you're describing would effectively happen on any other theme. As a general note, it might be a good idea to ask the maintainers of mike what they suggest (after searching their issue tracker for similar requests). Finally, if somebody has a good recipe how this can be built of, feel free to share ☺️

**Reply by berkantsahn** [reply]
Thanks for reply. If anyone has had this problem before and solved it, I'd be very grateful. I am also working on the subject, if i can solve it i will update the topic

**Reply by afnanshirol** [reply]
@berkantsahn is this issue resolved?  Did you find any solution or workaround?
